lib/arguments/file-url.js

```diff
@@ -4,7 +4,10 @@ import {fileURLToPath} from 'node:url';
 export const safeNormalizeFileUrl = (file, name) => {
 	const fileString = normalizeFileUrl(file);
 
-	if (typeof fileString !== 'string') {
+	const isString = typeof fileString === 'string'
+		|| fileString?.__proto__ === String.prototype; // eslint-disable-line no-proto
```

Comment:
This needs a short comment that it's a Deno workaround.

Reply:
Updated.